zhaquirks/ikea/__init__.py

```diff
@@ -73,6 +82,94 @@ class ScenesCluster(CustomCluster, Scenes):
     )
 
 
+class ShortcutV1Cluster(EventableCluster):
+    """Ikea Shortcut Button Cluster Variant 1."""
+
+    name = "ShortCutV1EventsCluster"
```

Comment:
Is this name used for anything?

(also -> `Shortcut`)
(and does it need to differ from the class name? I think the clusters UI always uses the class name at the moment?)

Reply:
K.a.
I was copy it from the original V1 PR with all its tricks was working OK so i have not changing it.